benchmark/buffers/dataview-set.js

```diff
@@ -41,17 +41,19 @@ function main(conf) {
 
 function benchInt(dv, fn, len, le) {
   var m = mod[fn];
+  var method = dv[fn];
```

Comment:
Well, if changing only those to `const` and not touching `var i` -> `let i`, then it looks fine, within the limtis.